provisioning/service/src/Models/EnrollmentGroup.cs

```diff
@@ -122,86 +123,10 @@ public EnrollmentGroup(string enrollmentGroupId, Attestation attestation)
-        /// <code>
-        /// {
-        ///    "enrollmentGroupId":"validEnrollmentGroupId",
-        ///    "attestation":{
-        ///        "type":"x509",
-        ///        "signingCertificates":{
-        ///            "primary":{
-        ///                "certificate":"[valid certificate]",
-        ///                "info": {
-        ///                    "subjectName": "CN=ROOT_00000000-0000-0000-0000-000000000000, OU=Azure IoT, O=MSFT, C=US",
-        ///                    "sha1Thumbprint": "0000000000000000000000000000000000",
-        ///                    "sha256Thumbprint": "validEnrollmentGroupId",
-        ///                    "issuerName": "CN=ROOT_00000000-0000-0000-0000-000000000000, OU=Azure IoT, O=MSFT, C=US",
-        ///                    "notBeforeUtc": "2017-11-14T12:34:18Z",
-        ///                    "notAfterUtc": "2017-11-20T12:34:18Z",
-        ///                    "serialNumber": "000000000000000000",
-        ///                    "version": 3
-        ///                }
-        ///            }
-        ///        }
-        ///    },
-        ///    "iotHubHostName":"ContosoIoTHub.azure-devices.net",
-        ///    "provisioningStatus":"enabled",
-        ///    "createdDateTimeUtc": "2017-09-28T16:29:42.3447817Z",
-        ///    "lastUpdatedDateTimeUtc": "2017-09-28T16:29:42.3447817Z",
-        ///    "etag": "\"00000000-0000-0000-0000-00000000000\""
-        /// }
-        /// </code>
-        /// </example>
-        /// <param name="enrollmentGroupId">The string with a unique id for the enrollmentGroup. It cannot be null or empty.</param>
-        /// <param name="attestation">The <see cref="AttestationMechanism"/> for the enrollment. It shall be `X509` or `SymmetricKey`.</param>
-        /// <param name="iotHubHostName">The string with the target IoT hub name. This is optional and can be null or empty.</param>
-        /// <param name="initialTwinState">The <see cref="TwinState"/> with the initial Twin condition. This is optional and can be null.</param>
-        /// <param name="provisioningStatus">The <see cref="ProvisioningStatus"/> that determine the initial status of the device. This is optional and can be null.</param>
-        /// <param name="createdDateTimeUtc">The DateTime with the date and time that the enrollment was created. This is optional and can be null.</param>
-        /// <param name="lastUpdatedDateTimeUtc">The DateTime with the date and time that the enrollment was updated. This is optional and can be null.</param>
-        /// <param name="eTag">The string with the eTag that identify the correct instance of the enrollment in the service. It cannot be null or empty.</param>
-        /// <param name="capabilities">The capabilities of the device (ie: is it an edge device?)</param>
-        /// <exception cref="DeviceProvisioningServiceException">If the received JSON is invalid.</exception>
-        [JsonConstructor]
-        internal EnrollmentGroup(
```

Comment:
We only need an empty ctor to make serialization work. I don't know why we had this uber ctor, but it isn't referenced anywhere so it seems safe to replace with an empty ctor